data/json/requirements/melee.json

```
@@ -0,0 +1,682 @@
+        [ "ch_knife_rambo", -1 ],
+        [ "qt_knife_rambo", -1 ],
+        [ "knife_rm42", -1 ],
+        [ "knife_folding", -1 ],
+        [ "knife_trench", -1 ],
+        [ "lc_knife_trench", -1 ],
+        [ "mc_knife_trench", -1 ],
+        [ "hc_knife_trench", -1 ],
+        [ "ch_knife_trench", -1 ],
+        [ "qt_knife_trench", -1 ],
+        [ "knife_baselard", -1 ],
+        [ "kris", -1 ],
+        [ "kukri", -1 ],
+        [ "lc_kukri", -1 ],
+        [ "mc_kukri", -1 ],
+        [ "ch_kukri", -1 ],
+        [ "hc_kukri", -1 ],
+        [ "qt_kukri", -1 ],
+        [ "tanto", -1 ],
+        [ "karambit", -1 ],
+        [ "kirpan", -1 ],
+        [ "butterfly_swords", -1 ]
+      ]
+    ]
+  },
+  {
+    "id": "basic_thrusting_swords",
+    "type": "requirement",
+    "//": "Any weapon that you could thrust with and stab someone.",
+    "tools": [
+      [
+        [ "sword_metal_fencing", -1 ],
+        [ "estoc_fake", -1 ],
+        [ "estoc_inferior", -1 ],
+        [ "fencing_foil", -1 ],
+        [ "fencing_foil_sharpened", -1 ],
+        [ "shock_foil", -1 ],
+        [ "fencing_epee", -1 ],
+        [ "fencing_epee_sharpened", -1 ],
+        [ "shock_epee", -1 ],
+        [ "fencing_sabre", -1 ],
+        [ "fencing_sabre_sharpened", -1 ],
+        [ "shock_sabre", -1 ],
+        [ "rapier_fake", -1 ],
+        [ "sharpened_pipe", -1 ]
+      ]
+    ]
+  },
+  {
+    "id": "real_thrusting_swords",
+    "type": "requirement",
+    "//": "Any real, quality weapon designed to be used in a thrusting fashion.  Needs to be a sword, not a spear.  This includes fencing weaponry.",
+    "tools": [
+      [
+        [ "estoc", -1 ],
+        [ "lc_estoc", -1 ],
+        [ "mc_estoc", -1 ],
+        [ "ch_estoc", -1 ],
+        [ "hc_estoc", -1 ],
+        [ "qt_estoc", -1 ],
+        [ "sword_cane", -1 ],
+        [ "rapier", -1 ],
+        [ "lc_rapier", -1 ],
+        [ "hc_rapier", -1 ],
+        [ "ch_rapier", -1 ],
+        [ "mc_rapier", -1 ],
+        [ "qt_rapier", -1 ],
+        [ "lc_talwar", -1 ],
+        [ "mc_talwar", -1 ],
+        [ "hc_talwar", -1 ],
+        [ "ch_talwar", -1 ],
+        [ "qt_talwar", -1 ],
+        [ "cavalry_sabre", -1 ],
+        [ "lc_cavalry_sabre", -1 ],
+        [ "mc_cavalry_sabre", -1 ],
+        [ "hc_cavalry_sabre", -1 ],
+        [ "ch_cavalry_sabre", -1 ],
+        [ "qt_cavalry_sabre", -1 ],
+        [ "jian", -1 ],
+        [ "lc_jian", -1 ],
+        [ "mc_jian", -1 ],
+        [ "hc_jian", -1 ],
+        [ "ch_jian", -1 ],
+        [ "qt_jian", -1 ],
+        [ "broadsword", -1 ],
+        [ "lc_broadsword", -1 ],
+        [ "mc_broadsword", -1 ],
+        [ "ch_broadsword", -1 ],
+        [ "hc_broadsword", -1 ],
+        [ "qt_broadsword", -1 ]
+      ]
+    ]
+  },
+  {
+    "id": "basic_short_swords",
+    "type": "requirement",
+    "//": "Any basic thing you could use as a short sword, exlcudes high-quality things.",
+    "tools": [ [ [ "null", -1 ] ] ]
```

```suggestion
    "tools": [ [ [ "makeshift_machete", -1 ], [ "machete_gimmick", -1 ], [ "machete", -1 ] ] ]
```
Why not add the normal and improvised machetes here?